### ISSUE TITLE
Non-interactive mode

### ISSUE THREAD

**Issue by copyme** [tmontaigu/CloudCompare-PythonPlugin#36]
Hello!

Is there a way to avoid interaction? For example, is there a way to save a PLY file while avoiding the window, which asks for the data type (BINARY vs. ASCII)?

Best,
Kacper

**Comment by tmontaigu** [comment]
Yep you have to do it this way, since this plugins only exposes cloudcompare's function as they are, if a dialog popup, then you can only make it not pop up if the function has a parameter that disables it
